src/lib/components/Footer.svelte

```diff
@@ -0,0 +1,49 @@
+<script type="ts">
+  import { FOOTER_ELEMENTS } from "$lib/config/homePageData";
+</script>
+
+<section class="surface-tint">
+  <div class="container">
+    <div class="footer-line" />
+    <div class="footer-links on-primary-text">
+      {#each FOOTER_ELEMENTS as footElements}
+        <a class="inverse-on-surface-text" href={footElements.url}
+          >{footElements.name}</a
+        >
+      {/each}
+    </div>
+  </div>
+</section>
+
+<style lang="scss">
+  .container {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    max-width: 50%;
```

Comment:
This max-width is not wrong but I believe there is a better way of doing this.

Here, what you want to make is a narrower container. 
But to make the whole page more consistent and with better alignment is good to constrain the possible container widths to just a few.

That's why if you look in `_containers.scss` you'll see we have `.container-small`, `.cotainer-medium` etc.
This way instead of having many different widths of the container that looks disordered and misaligned we always try to fit stuff to a constrained amount of widths, making the page look very aligned and clean.